stetho/src/main/java/com/facebook/stetho/inspector/helper/DocumentAccessRegisteredListener.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.helper;
+
+import com.facebook.stetho.inspector.elements.Document;
+
+/*
```

Comment:
This class only does 1 little thing for clients ... it calls `addRef` and `release`. Do we need to put this into a base class? Seems like `DOM` and `CSS` could probably handle this by themselves without being required to derive their `PeersRegisteredListener` from this?